packages/auth/src/grant/routes.ts

```diff
@@ -212,17 +295,46 @@ async function continueGrant(
     interactionService
   } = deps
 
+  if (!ctx.request.body || Object.keys(ctx.request.body).length === 0) {
+    await pollGrantContinuation(deps, ctx, continueId, continueToken)
+    return
+  }
+
+  const { interact_ref: interactRef } = ctx.request.body
+  if (!interactRef) {
+    ctx.throw(401, {
+      error: {
+        code: 'invalid_request',
+        description: 'missing interaction reference'
+      }
+    })
+  }
+
   const interaction = await interactionService.getByRef(interactRef)
   if (
     !interaction ||
     !isContinuableGrant(interaction.grant) ||
     !isMatchingContinueRequest(continueId, continueToken, interaction.grant)
   ) {
-    ctx.throw(404, { error: 'unknown_request' })
+    ctx.throw(404, {
+      error: { code: 'unknown_request', description: 'grant not found' }
```

Comment:
Should this also be `invalid_continuation`? or maybe `invalid_request` (to pick something from the list of codes?

Reply:
Will use `invalid_continuation`, I feel it fits best given the text quoted in https://github.com/interledger/rafiki/pull/2285#discussion_r1452857289